configuration/scripts/options/set_nml.jra55_2008

```diff
@@ -9,7 +9,7 @@ maskhalo_dyn   = .true.
 maskhalo_remap = .true.
 maskhalo_bound = .true.
 fyear_init      = 2008
-atm_data_dir    = 'ICE_MACHINE_INPUTDATA/CICE_data/forcing/gx1/JRA55'
+atm_data_dir    = '/p/work1/allard/CICE_data/forcing/gx1/JRA55'
```

Comment:
What's happening here?  We cannot have things pointing to a user space.  We should make sure there are no other dirs set like this.

Reply:
I made changes to remove references to /p/work1/allard; files now point to "ICE_MACHINE_INPUTDATA"